### ISSUE TITLE
Release command takes forever

### ISSUE THREAD

**Issue by imranismail** [bitwalker/distillery#519]
### Steps to reproduce

`mix release` under dev or prod

I was at 2.0.5 and after updating to 2.0.9 this happens.

### Verbose Logs

![](https://i.imgur.com/Vh4B8dU.png)

### Description of issue

- What are the expected results?
Shouldn't take too long
- What version of Distillery?
2.0.9
- What OS, Erlang/Elixir versions are you seeing this issue on?
Debian and MacOS

**Comment by bitwalker** [comment]
I'm not able to reproduce this - can you provide any additional information or a reproducible case? Are you able to reproduce with my [test project](https://github.com/bitwalker/distillery-test)?

**Comment by danschultzer** [comment]
After upgrading from `2.0.0-rc.4` to `2.0.9` I've experienced that building releases takes very long time too (went from around 20-ish seconds to 500+ seconds, that caused timeouts in our CI). It's a big complex project so it'll be difficult to replicate. Instead I'll try dig through what change in distillery that caused the increase in build time.

I did some testing, and it seems that reverting to `2.0.5` fixes the issue, there's some other issues going on with `2.0.7`-`2.0.8` (seeing some `Invalid start type given for` errors).
